src/SDL_internal.h

```diff
@@ -263,6 +263,14 @@
 extern "C" {
 #endif
 
+#if !defined(SDL_RELEASE_BUILD) || !defined(SDL_DEBUG_BUILD)
```

Comment:
```suggestion
#if !defined(SDL_RELEASE_BUILD) && !defined(SDL_DEBUG_BUILD)
```
It seems one of these is expected to be defined, so probably `&&` was meant here?